ansible/roles/flannel/files/40-docker-depends-on-flannel.conf

```diff
@@ -0,0 +1,9 @@
+# Dropin for docker.service
+
+[Unit]
+# Flannel must be started before Docker.
```

Comment:
@danehans I added links on bug-report you found and CoreOS documentation from which I learned this.